app/src/main/res/values-bg/strings.xml

```diff
@@ -19,7 +19,7 @@
     <string name="pin_conversation">Закачване отгоре</string>
     <string name="unpin_conversation">Откачване</string>
     <string name="forward_message">Препращане</string>
-    <string name="compress_error">Невъзможно е да се компресира изображението до избрания размер</string>
+    <string name="compress_error">Unable to compress image to selected size</string>
```

Comment:
why are you reverting all these translations?

Reply:
sorry, my bad, did not notice it. 
Reverted those files :+1: